semeio/workflows/localisation/local_script_lib.py

```diff
@@ -236,21 +236,24 @@ def write_scaling_values(
         raise KeyError(f" Method name: {method_name} is not implemented")
 
     scaling_values = np.zeros((nx, ny, nz), dtype=np.float32)
-    for k, j, i in itertools.product(nz, ny, nx):
-        global_index = grid_for_field.global_index(ijk=(i, j, k))
-        scaling_values[i, j, k] = decay_obj(global_index)
+    for k in range(nz):
+        for j in range(ny):
+            for i in range(nx):
+                global_index = grid_for_field.global_index(ijk=(i, j, k))
+                scaling_values[i, j, k] = decay_obj(global_index)
 
     scaling_kw_name = "S_" + str(scaling_parameter_number)
     scaling_kw = grid_for_field.create_kw(scaling_values, scaling_kw_name, False)
-    filename = corr_name + "_" + node_name + "_scaling" + ".GRDECL"
-    print(f"   -- Write file: {filename}")
+    filename = corr_name + "_" + node_name + "_" + scaling_kw_name + ".GRDECL"
+    debug_print(f"Write file: {filename}", LogLevel.LEVEL3)
     with cwrap.open(filename, "w") as file:
         grid_for_field.write_grdecl(scaling_kw, file)
 
 
 def activate_and_scale_correlation(
     grid, method, row_scaling, ref_pos, data_size, main_range, perp_range, azimuth
 ):
+    debug_print(f"Scale correlations using method: {method}", LogLevel.LEVEL3)
```

Comment:
Do we really need all these `print` statements? I get why some of them are important, but are all of them equally relevant?

Reply:
I prefer to have sufficent prints to be able to see all essential steps in the output log, but it is easy to turn off by adjusting log level.